### PR TITLE
Check null before creating $range

### PR DESCRIPTION
**Pull request by muglug**
I'm not sure this will have much of an effect, but every little helps...

### CODE REVIEW

**Comment by felixfbecker** [comment]
There is also `NodeTraverser::STOP_TRAVERSAL` now

**Comment by muglug** [comment]
Ah right! Want me to update? Or will you implement?

**Comment by felixfbecker** [comment]
Feel free :)

File: src/NodeVisitor/NodeAtPositionFinder.php [hunk 1]
```diff
@@ -34,9 +34,12 @@ public function __construct(Position $position)
...
     public function leaveNode(Node $node)
     {
-        $range = Range::fromNode($node);
-        if (!isset($this->node) && $range->includes($this->position)) {
-            $this->node = $node;
+        if ($this->node === null) {
+            $range = Range::fromNode($node);
+            if ($range->includes($this->position)) {
+                $this->node = $node;
+                return NodeTraverser::STOP_TRAVERSAL
```
Comment (felixfbecker) [inline src/NodeVisitor/NodeAtPositionFinder.php]: missing semicolon

Comment (muglug) [inline src/NodeVisitor/NodeAtPositionFinder.php]: Sorry, deeply embarrassing...

**Pull request merged**